README.md

```diff
@@ -7,6 +7,20 @@ crowdsourcing workers, and from those judgments, to compute relevance metrics
 such as normalized discounted cumulative gain (NDCG) and mean average precision
 (MAP).
 
+## Running tests
+
+From a fresh arcs virtual environment, install pytest:
+
+```bash
+pip install pytest
```

Comment:
Oh. Yeah. I think we should change that per the answer here:

http://stackoverflow.com/questions/15422527/best-practices-how-do-you-list-required-dependencies-in-your-setup-py

Cool?